mt-kahypar/partition/mapping/target_graph.h

```diff
@@ -299,9 +299,39 @@ class TargetGraph {
     return 0;
   }
 
+
+  HyperedgeWeight distance(const ds::Bitset& ) const {
+    return 0;
+  }
+
+  HyperedgeWeight distance(const PartitionID , const PartitionID ) const {
```

Comment:
Formatting (also apply to other functions)
```suggestion
  HyperedgeWeight distance(const PartitionID, const PartitionID) const {
```